site/en/docs/extensions/mv3/getstarted/tut-reading-time/index.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Reading time'
+description: 'Learn how to insert an element on each page.'
+subhead: 'Create your first extension that inserts a new element on the page.'
+date: 2022-08-10
+# updated: 2022-06-13
+---
+
+## Overview {: #overview }
+
+This tutorial builds an extension that adds the expected reading time to any Chrome extension and
+Chrome web store documentation page. 
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/VczSGe8eh0Xv7nTXxhxg.png", 
+alt="Reading time extension in the extension Welcome page", width="500", height="116", class="screenshot" %}
+  <figcaption>
+  Reading time extension on the extension's Welcome page. 
+  </figcaption>
+</figure>
+
+In this guide, we’re going to explain the following concepts:
+
+- The extension manifest.
+- What icon sizes an extension uses.
+- How to inject code on pages using [content scripts][doc-cs].
+- How to use match patterns.
+- Extension permissions
```

Comment:
As I recall, we've previously discussed having this kind of list link out to other documents, but were worried that doing so would distract readers. What do you think about having a list of links at the end of the article? Maybe the section could be titled something like "Related topics", "Additional articles", or "Keep reading".

Reply:
I'd like to explore this more, let's brainstorm a bit more about this and what else could go in this section

Reply:
Two thoughts:
1. We have links to all these resources throughout the tutorial.
2. We added a "Continue exploring" section at the end of each tutorial.

I think this might be a bit redundant at this point, but I'm open to talking more about it with the team.